runtime/metrics.go

```diff
@@ -47,7 +47,7 @@ func newMetrics(re *RunEnv) *Metrics {
 	}
 
 	m.diagnostics = newMetricsApi(re, metricsApiOpts{
-		freq:        5 * time.Second,
```

Comment:
I think 5 seconds is a bit aggressive, especially when we want to run 10k instances. We should probably think about making this configurable, but for now 10sec. or 30sec. is good enough.

Reply:
You are right, I don't mind reverting to 5sec, I just think we are wasting disk space, because the 10sec. granularity has appeared to work well for me for now ;)

100 measurements * 10sec. * 10000 node testplans is a lot of data on disk, specially if we want to do longer retention in the future.

Reply:
Sure, but we shouldn't assume 10k is the _default_ size of tests. Users can easily change the frequency by calling `runenv.D().SetFrequency()` for larger tests. Let's let the user take the tradeoff they want.